safe/utilities/utilities.py

```diff
@@ -412,3 +414,13 @@ def readable_os_version():
         return ' {version}'.format(version=platform.mac_ver()[0])
     elif platform.system() == 'Windows':
         return platform.platform()
+
+def is_plugin_installed(name):
+    """Check if a plugin is installed, even if it's not enabled.
+
+    :param name: Name of the plugin to check.
+    :type name: string
+    """
+    directory = QgsApplication.qgisSettingsDirPath()
+    path = join(directory, 'python', 'plugins', name)
+    return isdir(path)
```

Comment:
__[flake8]__
* [W292] no newline at end of file

<!-- Comment by [SideCI](https://sideci.com) -->